### ISSUE TITLE
Preserve code folding

### ISSUE THREAD

**Issue by stdavis** [lexicalunit/atom-isort#19]
Thanks for the great plugin. I love not having to worry about keeping my imports organized.

One enhancement request:

I wish that this tool preserved any code folding that I have done when it runs. For example. 

I have "Sort On Save" enabled.
I start working on a file and folder some code.
When I save the file, all of the folded code becomes unfolded and I have to refold it.

**Comment by lexicalunit** [comment]
Yep, that would be nice to have; I wasn't even aware it did that! Seems like it should be possible.
